### PR TITLE
docs: add deployment info for Qovery

### PR DESCRIPTION
**Pull request by arnaudjnn**
Hi Nuxt.js team,

I added docs on deploying Nuxt.js to Qovery - feedback is welcome.

Arnaud.

### CODE REVIEW

File: content/en/guides/deployment/deployment-qovery.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Deploy Nuxt on Qovery
+description: How to deploy Nuxt.js on Qovery?
+menu: Qovery
+target: Server
```
Comment (danielroe) [inline content/en/guides/deployment/deployment-qovery.md:5]: should this be Static & Server?
```suggestion
target: Static & Server
```

File: content/en/guides/deployment/deployment-qovery.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- Continuous, automatic builds & deploys from GitHub, Bitbucket, and GitLab.
+- Automatic SSL certificates through [Let's Encrypt](https://letsencrypt.org).
+- Free managed PostgreSQL.
+- Free SSD storage.
+- Unlimited collaborators.
+- Unlimited [custom domains](https://docs.qovery.com/guides/getting-started/setting-custom-domain/).
+
+## Prerequisites
+
+This guide assumes you already have a Nuxt project to deploy. If you need a project, use the [Get Started](/content/en/get-started/installation.md).
```
Comment (danielroe) [inline content/en/guides/deployment/deployment-qovery.md:23]:
```suggestion
This guide assumes you already have a Nuxt project to deploy. If you need a project, follow the [Get Started](/content/en/get-started/installation.md) guide.
```

**Review by danielroe — CHANGES_REQUESTED** [review]
Just picking up on this again - looks great, just needs a few tweaks.

**Comment by evoxmusic** [comment]
Hi @danielroe , @arnaudjnn will apply your changes on Monday :)

**Comment by arnaudjnn** [comment]
> Just picking up on this again - looks great, just needs a few tweaks.

I made the changes as required. Let me know

File: content/en/guides/deployment/deployment-qovery.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Continuous, automatic builds & deploys from GitHub and GitLab.
+- Automatic SSL certificates through [Let's Encrypt](https://letsencrypt.org).
+- Free managed PostgreSQL.
+- Free SSD storage.
+- Unlimited collaborators.
+- Unlimited [custom domains](https://docs.qovery.com/guides/getting-started/setting-custom-domain/).
+
+## Prerequisites
+
+This guide assumes you already have a Nuxt project to deploy. If you need a project, follow the [Get Started](/content/en/get-started/installation.md) guide.
```
Comment (danielroe) [inline content/en/guides/deployment/deployment-qovery.md:23]:
```suggestion
This guide assumes you already have a Nuxt project to deploy. If you need a project, follow the [Get Started](/docs/2.x/get-started/installation) guide.
```

File: content/en/guides/deployment/deployment-qovery.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Setup
+
+Follow the procedure below to set up a Nuxt on Qovery:
```
Comment (danielroe) [inline content/en/guides/deployment/deployment-qovery.md:27]:
```suggestion
Follow the procedure below to set up Nuxt on Qovery:
```

**Review by danielroe — APPROVED** [review]

**Pull request merged**